analysisservice/src/test/java/nu/yona/server/analysis/service/AnalysisEngineServiceTests.java

```diff
@@ -244,15 +275,31 @@ public void conflictInterval()
 		service.analyze(userAnonId, new NetworkActivityDto(conflictCategories, "http://localhost/test1", Optional.empty()));
 
 		// Verify that there is a new conflict message sent.
-		ArgumentCaptor<MessageDestinationDto> messageDestination = ArgumentCaptor.forClass(MessageDestinationDto.class);
```

Comment:
Yes, sorry. At the time I refactored this test case, I didn't see the helper methods yet. After I saw them, I didn't go back. Corrected it now.